.github/workflows/gqcp.yml

```diff
@@ -0,0 +1,114 @@
+name: GQCP
+
+on: [pull_request]
+
+# These variables are for the slack notification, the default is failure, if all build steps are successful they will be updated to green and "Build successful"
```

Comment:
```suggestion
```

Are these variables defined in the same environment as the ones in jobs:build:env?

Reply:
I actually didn't test if it would work that way.